api/shapes/loadouts.ts

```diff
@@ -12,6 +12,12 @@ export interface LoadoutItem {
    * (by it's hash) is supposed to be socketed into the given socket index.
    */
   socketOverrides?: { [socketIndex: number]: number };
+  /**
+   * UTC epoch milliseconds timestamp of when the item was crafted. Used to
+   * match up items that have changed instance ID from being reshaped since they
+   * were added to the loadout.
+   */
+  dateCrafted?: number;
```

Comment:
what i would like to store in DIM API is the **canonical** timestamp (UTC seconds).
seems more cross-app compatible. i would be fine with changing DimCrafted.dateCrafted to be that as well. it's currently used a grand total of no places in DIM, and can be x1000'd and Date('d at render/display time, instead of while creating a DimItem

Reply:
Sure. I went ahead and renamed it to a less awkward name too.